package.json

```diff
@@ -35,7 +37,14 @@
     "eslint-config-screwdriver": "^2.0.9",
     "jenkins-mocha": "^3.0.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "hapi": "^16.1.0",
+    "joi": "^10.2.2",
+    "mockery": "^2.0.0",
+    "nodemailer": "^3.1.3",
+    "sinon": "^1.17.7",
```

Comment:
`npm install -S <module_name>` will install the module and save it in the "dependencies" list.

`npm install -D <module_name>` will install the module and save it in the "devDependencies" list